spec/lib/pdf_fill/filler_spec.rb

```diff
@@ -109,7 +109,7 @@ def compare_pdfs(pdf1, pdf2)
   end
 
   describe '#fill_ancillary_form', run_at: '2017-07-25 00:00:00 -0400' do
```

Comment:
this whole method can be refactored out, not sure why it was added

Reply:
i think the code could have been written so that `fill_ancillary_form` was not added and the original `fill_form` was used instead, i agree it's outside the scope of this pr though

Reply:
Ah, yes, I agree and I'm not sure why this was added.